pytket/docs/index.rst

```diff
@@ -87,7 +87,7 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     :caption: More Documentation:
     :maxdepth: 1
     
-    Manual <https://cqcl.github.io/pytket/manual/index.html>
+    Manual <https://tket.quantinuum.com/user-manual/>
     Extensions <https://cqcl.github.io/pytket-extensions/api/index.html>
```

Comment:
I think we could replace this link as well?

Reply:
Ah yes, just had some merge conflicts to resolve. 

Done now :)